src/components/AllArticles.jsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "react"
+import { getArticles } from "../api";
+import { Link } from "react-router-dom";
+
+
+
+const AllArticles = ( )=> {
+    const [articles, setArtcicles] = useState ([]);
+
+    useEffect(()=> {
+        getArticles().then((res)=> {
+            setArtcicles(res);
+        });
+}, [])
+
+
+return (
+
+    <main className="AllArt">
```

Comment:
An is loading design pattern would be useful here to help the user see that data is being loaded.